### ISSUE TITLE
(NOT "SKEET") To make Bluesky different from Twitter, a fundamental review is necessary. First the name itself, change “mute” to “hide”.

### ISSUE THREAD

**Issue by lamrongol** [bluesky-social/social-app#867]
The word "mute" is not originally used to a person, and the act of "mute" someone gives an inhumane impression.
How about simply naming it "hide"?

In addition, while "Report" should remain and "Public Block" is good new function, simple "block" function should rather be removed.
Because if an evil person sends a lot of replies to someone for harassment, and there is no simple "block" function but only a "hide" function on Bluesky, they will probably think like this: "Does this person actually 'hide' me? If I'm blocked on Twitter, I can know it. But Bluesky only has a 'hide' function, so I can't know what this person is doing. I'm sending a lot of replies for harassment, but maybe they're not even reaching this person. What I'm doing might be completely futile". Then they will feel frustrated and leave.
One may think, "Without a (simple) block function, my posts would be seen". However, unless it is a private account, posts can be seen if evil people log out from blocked accounts, so this is a completely meaningless specification.

**Comment by lamrongol** [comment]
> Blocks also stop interactions which makes it harder for harassers to send other people to harass a user.

A simple "block" rather than a "block list" will not stop harassers to send other people.
Please reopen.
If that is not possible, I will create a new Issue with only the suggestion to change the name "mute" to "hide".
